lib/puppet-lint/bin.rb

```diff
@@ -78,11 +93,45 @@ def run
       end
       puts ']' if PuppetLint.configuration.json
 
+      report_sarif(all_problems, full_base_path, full_base_path_uri) if PuppetLint.configuration.sarif
+
       return return_val
     rescue PuppetLint::NoCodeError
       puts 'puppet-lint: no file specified or specified file does not exist'
       puts "puppet-lint: try 'puppet-lint --help' for more information"
       return 1
     end
   end
+
+  # Internal: Print the reported problems in SARIF format to stdout.
+  #
+  # problems - An Array of problem Hashes as returned by
+  #            PuppetLint::Checks#run.
+  #
+  # Returns nothing.
+  def report_sarif(problems, base_path, base_path_uri)
+    sarif_file = File.read(File.join(File.dirname(File.expand_path(__FILE__)), 'report/sarif.json'))
```

Comment:
Isn't this the same?
```suggestion
    sarif_file = File.read(File.join(__dir__, 'report', 'sarif.json'))
```

Reply:
fixed